docs/requirements.txt

```diff
@@ -0,0 +1,4 @@
+Sphinx>=2.1.2
```

Comment:
Can you add/update these in `setup.cfg` under the `doc` extra?

https://github.com/poldracklab/smriprep/blob/5b519b1e2446ec8b69848575d7bc67800474f035/setup.cfg#L48-L54

Reply:
A useful tool we could include here, to keep things synced: https://github.com/nipy/nibabel/blob/master/tools/update_requirements.py

We could update it to also write out to `docs/requirements.txt`.